src/main/java/sirius/biz/storage/layer2/URLBuilder.java

```diff
@@ -179,6 +179,7 @@ public URLBuilder withVariant(String variant) {
      * @param path the filename to send to the browser
      * @return the builder itself for fluent method calls
```

Comment:
You can comment in the`@deprecated` javadoc which method(s) the user shall call instead.